diff --git a/android/CMakeLists.txt b/android/CMakeLists.txt
index d4731e6c9e5b7cfee8c14dcd7c34639e4d3842d6..8b720bb1fb02b5091bdd1916bad5b66569cf121e 100644
--- a/android/CMakeLists.txt
+++ b/android/CMakeLists.txt
@@ -86,4 +86,4 @@ set_target_properties(
   worklets
   PROPERTIES
     IMPORTED_LOCATION
-    "${REACT_NATIVE_WORKLETS_DIR}/android/build/intermediates/cmake/${BUILD_TYPE}/obj/${ANDROID_ABI}/libworklets.so"
+    "${REACT_NATIVE_WORKLETS_DIR}/android/build/intermediates/prefab_package/${BUILD_TYPE}/prefab/modules/worklets/libs/android.${ANDROID_ABI}/libworklets.so"
